packages/typescript/test/lit_plugin/BUILD.bazel

```diff
@@ -0,0 +1,36 @@
+# Copyright 2017 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("@npm_bazel_typescript//:index.from_src.bzl", "ts_library")
+
+ts_library(
+    name = "lit_plugin",
+    srcs = glob(["*.ts"]),
+    expected_diagnostics = [
+        """TS2322: \\[lit\\] Unknown tag "unknown-element". Did you mean 'lit-element'?""",
+        "TS2322: \\[lit\\] Type '222' is not assignable to 'string",
+        "TS2322: \\[lit\\] Type '333' is not assignable to 'string",
+        """TS2322: \\[lit\\] Type '"aaa"' is not assignable to 'number'""",
+        #"""TS2322: \\[lit\\] Type '"bbb"' is not assignable to 'number'""",
```

Comment:
nit: why the commented out assertion?

Reply:
@rictic might TAL at at - I couldn't get this one to trigger. Not really important though, hard to imagine there's a bug in how we invoke the plugin, since we just pass it the whole options block. Will remove